eo-maven-plugin/src/main/java/org/eolang/maven/DcsWithRuntime.java

```diff
@@ -23,47 +23,107 @@
  */
 package org.eolang.maven;
 
+import com.jcabi.xml.XMLDocument;
+import java.io.IOException;
+import java.net.URL;
 import java.util.Iterator;
 import org.apache.maven.model.Dependency;
 import org.cactoos.list.ListOf;
+import org.cactoos.scalar.Sticky;
+import org.cactoos.scalar.Unchecked;
 
 /**
  * Add runtime dependency if it is absent.
  *
  * @since 0.28.11
- * @todo #1361:90min Hardcoded version of EoRuntimeDependency.
- *  See the EoRuntimeDependency constructor for more info.
- *  It's much better to determine the version of the runtime library
- *  dynamically. For example, we can fetch the latest version by http
- *  or from config files.
  */
 final class DcsWithRuntime implements Dependencies {
 
     /**
-     * The dependencies source.
+     * Dependency downloaded by HTTP from Maven Central.
+     */
+    private static final Unchecked<Dependency> FROM_MAVEN = fromMaven();
+
+    /**
+     * All dependencies.
      */
     private final Dependencies delegate;
 
     /**
-     * The main constructor.
+     * Runtime dependency source.
+     */
+    private final Unchecked<Dependency> source;
```

Comment:
@volodya-lombrozo let's rename to `fixed`

Reply:
@mximp renamed to `supplied`.